Cannnot gem install exif. Win 10, cant find libexif after pacman -S mingw-w64-x86_64-libexif make failed, exit code 2

## What problems are you experiencing?
Cannot `gem install exif` on Windows 10.0.19045

## Steps to reproduce
Install `rubyinstaller-devkit-3.2.2-1-x64.exe`
Run: `gem install exif`

I've also tried `ridk exec pacman -S mingw-w64-x86_64-libexif` then `gem install exif` & `gem install exif --platform ruby -- --use-system-libraries` with same outcome. How can I correctly specify the path to the lib so it will compile or fix whatever is broken? Thanks!

```
C:\Users\Brad Krane\Documents\src\exif>gem install exif
Building native extensions. This could take a while...
ERROR:  Error installing exif:
        ERROR: Failed to build gem native extension.

    current directory: C:/Ruby32-x64/lib/ruby/gems/3.2.0/gems/exif-2.2.4/ext/exif
C:/Ruby32-x64/bin/ruby.exe extconf.rb
checking for -lexif... no
checking for libexif/exif-data.h... no
checking for libexif >= 0.6.22... no
checking for libexif >= 0.6.23... no
checking for whether -D LIBEXIF_VERSION='"unknown"' is accepted as CFLAGS... yes
creating Makefile

current directory: C:/Ruby32-x64/lib/ruby/gems/3.2.0/gems/exif-2.2.4/ext/exif
make DESTDIR\= sitearchdir\=./.gem.20230813-29328-sitiig sitelibdir\=./.gem.20230813-29328-sitiig clean

current directory: C:/Ruby32-x64/lib/ruby/gems/3.2.0/gems/exif-2.2.4/ext/exif
make DESTDIR\= sitearchdir\=./.gem.20230813-29328-sitiig sitelibdir\=./.gem.20230813-29328-sitiig
generating exif-x64-mingw-ucrt.def
compiling data.c
data.c:4:10: fatal error: libexif/exif-data.h: No such file or directory
    4 | #include <libexif/exif-data.h>
      |          ^~~~~~~~~~~~~~~~~~~~~
compilation terminated.
make: *** [Makefile:248: data.o] Error 1

make failed, exit code 2

Gem files will remain installed in C:/Ruby32-x64/lib/ruby/gems/3.2.0/gems/exif-2.2.4 for inspection.
Results logged to C:/Ruby32-x64/lib/ruby/gems/3.2.0/extensions/x64-mingw-ucrt/3.2.0/exif-2.2.4/gem_make.out

C:\Users\Brad Krane\Documents\src\exif>ridk exec pacman -S mingw-w64-x86_64-libexif
resolving dependencies...
looking for conflicting packages...

Packages (1) mingw-w64-x86_64-libexif-0.6.24-2

Total Installed Size:  3.09 MiB

:: Proceed with installation? [Y/n] y
(1/1) checking keys in keyring                                                                                           [#######################################################################] 100%
(1/1) checking package integrity                                                                                         [#######################################################################] 100%
(1/1) loading package files                                                                                              [#######################################################################] 100%
(1/1) checking for file conflicts                                                                                        [#######################################################################] 100%
(1/1) checking available disk space                                                                                      [#######################################################################] 100%
:: Processing package changes...
(1/1) installing mingw-w64-x86_64-libexif                                                                                [#######################################################################] 100%

C:\Users\Brad Krane\Documents\src\exif>dir C:\Ruby32-x64\msys64\mingw64\include\libexif
 Volume in drive C has no label.
 Volume Serial Number is ECA4-34A2

 Directory of C:\Ruby32-x64\msys64\mingw64\include\libexif

2023-08-13  05:43    <DIR>          .
2023-08-13  05:43    <DIR>          ..
2022-02-12  01:35             1,563 exif-byte-order.h
2022-02-12  01:35             5,598 exif-content.h
2022-02-12  01:35             1,461 exif-data-type.h
2022-02-12  01:35             8,948 exif-data.h
2022-02-12  01:35             6,316 exif-entry.h
2022-02-12  01:35             1,981 exif-format.h
2022-02-12  01:35             1,675 exif-ifd.h
2022-02-12  01:35             4,301 exif-loader.h
2022-02-12  01:35             3,452 exif-log.h
2022-02-12  01:35             2,654 exif-mem.h
2022-02-12  01:35             4,333 exif-mnote-data.h
2022-02-12  01:35            11,334 exif-tag.h
2022-02-12  01:35             5,951 exif-utils.h
2022-02-12  01:35                76 _stdint.h
              14 File(s)         59,643 bytes
               2 Dir(s)  172,015,104,000 bytes free

C:\Users\Brad Krane\Documents\src\exif>gem install exif
Building native extensions. This could take a while...
ERROR:  Error installing exif:
        ERROR: Failed to build gem native extension.

    current directory: C:/Ruby32-x64/lib/ruby/gems/3.2.0/gems/exif-2.2.4/ext/exif
C:/Ruby32-x64/bin/ruby.exe extconf.rb
checking for -lexif... no
checking for libexif/exif-data.h... no
checking for libexif >= 0.6.22... no
checking for libexif >= 0.6.23... no
checking for whether -D LIBEXIF_VERSION='"unknown"' is accepted as CFLAGS... yes
creating Makefile

current directory: C:/Ruby32-x64/lib/ruby/gems/3.2.0/gems/exif-2.2.4/ext/exif
make DESTDIR\= sitearchdir\=./.gem.20230813-19312-ylqpfe sitelibdir\=./.gem.20230813-19312-ylqpfe clean

current directory: C:/Ruby32-x64/lib/ruby/gems/3.2.0/gems/exif-2.2.4/ext/exif
make DESTDIR\= sitearchdir\=./.gem.20230813-19312-ylqpfe sitelibdir\=./.gem.20230813-19312-ylqpfe
generating exif-x64-mingw-ucrt.def
compiling data.c
data.c:4:10: fatal error: libexif/exif-data.h: No such file or directory
    4 | #include <libexif/exif-data.h>
      |          ^~~~~~~~~~~~~~~~~~~~~
compilation terminated.
make: *** [Makefile:248: data.o] Error 1

make failed, exit code 2

Gem files will remain installed in C:/Ruby32-x64/lib/ruby/gems/3.2.0/gems/exif-2.2.4 for inspection.
Results logged to C:/Ruby32-x64/lib/ruby/gems/3.2.0/extensions/x64-mingw-ucrt/3.2.0/exif-2.2.4/gem_make.out
```


## What's the output from `ridk version`?
C:\Users\Brad Krane\Documents\src\exif>ridk version
---
ruby:
  path: C:/Ruby32-x64
  version: 3.2.2
  platform: x64-mingw-ucrt
  cc: gcc.exe (Rev10, Built by MSYS2 project) 12.2.0
ruby_installer:
  package_version: 3.2.2-1
  git_commit: 83df213
msys2:
  path: C:\Ruby32-x64\msys64
cc: gcc (Rev1, Built by MSYS2 project) 13.2.0
sh: GNU bash, version 5.2.15(1)-release (x86_64-pc-msys)
os: Microsoft Windows [Version 10.0.19045.3208]

C:\Users\Brad Krane\Documents\src\exif>ruby -v
ruby 3.2.2 (2023-03-30 revision e51014f9c0) [x64-mingw-ucrt]

--- a/exif.gemspec
+++ b/exif.gemspec
@@ -16,4 +16,5 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'rake', '>= 0.8.1'
   spec.add_development_dependency 'rake-compiler', '~> 1.0.4'
   spec.add_development_dependency 'minitest', '~> 5.11.3'
+  spec.metadata['msys2_mingw_dependencies'] = 'libexif'
 end

